Assets/Scripts/DaggerfallUnityApplication.cs

```diff
@@ -0,0 +1,107 @@
+// Project:         Daggerfall Unity
+// Copyright:       Copyright (C) 2009-2024 Daggerfall Workshop
+// Web Site:        http://www.dfworkshop.net
+// License:         MIT License (http://www.opensource.org/licenses/mit-license.php)
+// Source Code:     https://github.com/Interkarma/daggerfall-unity
+// Original Author: kaboissonneault
+// Contributors:    
+// 
+// Notes:
+//
+
+//#define SEPARATE_DEV_PERSISTENT_PATH
+
+using System;
+using System.IO;
+using UnityEngine;
+
+public static class DaggerfallUnityApplication
+{
+    static string persistentDataPath;
+    public static string PersistentDataPath { get { return persistentDataPath; } }
+
+    [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.SubsystemRegistration)]
+    static void SubsystemInit()
+    {
+#if UNITY_EDITOR && SEPARATE_DEV_PERSISTENT_PATH
+        persistentDataPath = String.Concat(Application.persistentDataPath, ".devenv");
+        Directory.CreateDirectory(persistentDataPath);
+#else
+        persistentDataPath = Application.persistentDataPath;
+#endif
+
+        InitLog();
+    }
+
+
+    public class LogHandler : ILogHandler, IDisposable
+    {
+        private FileStream fileStream;
+        private StreamWriter streamWriter;
+
+        public LogHandler()
+        {
+            string filePath = Path.Combine(PersistentDataPath, "Player.log");
+
+            try
+            {
+                if(File.Exists(filePath))
+                {
+                    string prevPath = Path.Combine(PersistentDataPath, "Player-prev.log");
+                    File.Delete(prevPath);
+                    File.Move(filePath, prevPath);
+                }
+            }
+            catch { }
```

Comment:
Not an approved reviewer so just throwing this out, but should this itself be logged? Not sure how much exception swallowing matters here, but I think it's best avoided in most cases.

Reply:
Done. 

I managed to test this by putting Player-prev.log as read-only. The prev log is not replaced, and Player.log gets started anew normally.

![image](https://github.com/Interkarma/daggerfall-unity/assets/5789925/90f55cf8-9cfd-40ae-8260-6e0107ccc789)